src/Microsoft.Cci.Extensions/Differs/Rules/TokenListDiffer.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+#if COREFX
+using System.Composition;
+#else
+using System.ComponentModel.Composition;
+#endif
+using System.Linq;
+using Microsoft.Cci.Writers;
+using Microsoft.Cci.Writers.Syntax;
+using Microsoft.Cci.Writers.CSharp;
+using Microsoft.Cci.Filters;
+
+namespace Microsoft.Cci.Differs.Rules
+{
+    [ExportDifferenceRule]
+    public class TokenListDiffer : DifferenceRule
+    {
+        [Import(AllowDefault = true)]
+#if COREFX
+        private IDiffingService _diffingService { get; set; }
```

Comment:
We should just switch this to a properly named property for both cases. Both versions of MEF support properties.